### ISSUE TITLE
[aliexpress.com] On Bundle deals, price should be written white and not black to be visible in dark mode

### ISSUE THREAD

**Issue by oeloo** [darkreader/darkreader#13292]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [X] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

The price is currently displayed in black color on a dark background. So it is not properly readable.
Please display the price in White color when in dark mode, to be able to read it on the dark background.

### Website Address

https://www.aliexpress.com/gcp/300000512/nnpage2024?disableNav=YES&pha_manifest=ssr&_immersiveMode=true&channelLinkTag=nn_newgcp

### Steps To Reproduce

1. Go to AliExpress.com , then to 'Bundle deals' section (or directly use this url https://www.aliexpress.com/gcp/300000512/nnpage2024?disableNav=YES&pha_manifest=ssr&_immersiveMode=true&channelLinkTag=nn_newgcp )
2. Look the price, they are hardly readable, because they are displayed in black color on a dark background.

### Screenshots

With Dark Reader enabled, the price is not readable:
![image](https://github.com/user-attachments/assets/8f99a6e7-e413-410f-848b-2b684675b511)

For information here is how it looks like without Dark Reader: the price tag is perfectly readable.
![image](https://github.com/user-attachments/assets/61458363-e92b-47d3-83f9-a78a5be00cd0)

### Operating System

Windows 11 (fully up to date)

### Web Browser name and version

Google Chrome (fully up to date)

### Dark Reader version

4.9.94

**Comment by corentinderuyck** [comment]
Hello, I just proposed a fix for your issue. Have a nice day!

**Comment by Myshor** [comment]
Sitefix added to Dark Reader.

- If you want to have this fix faster you can set **Synchronizee sites fixes** option to check in Advanced section of Settings.
![obraz](https://github.com/user-attachments/assets/9dea8b66-b77a-46ab-8aa6-e50d995718c3)
- With that enabled newest sitefixes will be updated every browser start. 😉
- To open all settings you must click on:
(Old/classic design) More -> All Settings
![obraz](https://github.com/user-attachments/assets/8c17b1f1-e85e-4672-8a88-85930e178e02)
(New/Preview design) Settings in popup
![obraz](https://github.com/user-attachments/assets/e4f586f8-8ce4-47bc-90d7-7451eb030439)

**Comment by oeloo** [comment]
The issue has come back again: the price is not readable because it is once again written in black on a black background
![image](https://github.com/user-attachments/assets/d4f23e62-aa6e-4b9a-835e-4b2c7182ca08)

**Comment by oeloo** [comment]
I have "Synchronize sites fixes" enabled. Also, this fix was working perfectly. I tested it just after your fix, and it was nice. But it is no longer working, though I have not changed anything in Chrome settings or my Chrome Extensions since that date. They do auto-update, though.

**Comment by Myshor** [comment]
I checked on Chromium Based browser to be sure and I can not reproduce either.

How can I try to help?

I have two things in mind that can help.

1. Open Dark Reader Developer Tools and click button "Reset Changes". It will force reset fixes to default (or newest if "Synchronize site fixes" is enabled).
![obraz](https://github.com/user-attachments/assets/ebe9c046-fade-4fad-b3d1-1d587a2fab81)
2. If this will not help removing whole extension from browser then add it back again with total reset of settings can help. Then you will need to reenable "Synchronize site fixes" back again by yourself.
3. If above 2 solutions won't help then come back with exact URL you have issue with prices which I can check on my side. (with link used from bug submission post I can not reproduce the problem).

**Comment by oeloo** [comment]
Thanks for your help. Resetting all dark settings and uninstalling/reinstalling Dark Reader did not fix the issue, either.
I have disabled all my Chrome addons one by one to find out, and the cause of the issue was:  ChatGPT for Google addon (https://chromewebstore.google.com/detail/chatgpt-for-google/jgjaeacdkonaoafenlfkkkmbaopkbilf?hl=en).
When I re-enable it, the issue comes back.
I do not use ChatGPT for Google addon on aliexpress.com, but it has access to 'On all sites' by default. Unfortunately, Chrome does seem to provide the possibility to blacklist a Chrome addon for a list of given sites (only a whitelist which is useless in this situation); I would have added aliexpress.com to tChatGPT for Google access blacklist.  As I need Dark Reader I had to completely disable ChatGPT for Google.

**Comment by Myshor** [comment]
Thanks for the feedback... Now I will need to have in mind that some other addon can break Dark Reader's functionality. Saving to my brain that ChatGPT for Google can be problematic. 🙂

**Comment by oeloo** [comment]
Yes, it is definitively causing issues on Dark Reader when the ChatGPT for Google add-on is just enabled. I was using all the default settings of ChatGPT, so repeating the issue should be straightforward.
The timing of issues corresponds also to ChatGPT for Google installation.
Given ChatGPT for Google is installed on 2 million Chrome browsers, it could cause issues for many Dark Reader users without realizing it. The most annoying is that the issues are not straightforward to notice, and it is causing issues on many other websites dark mode.
